src/mrdsrg-spin-integrated/mrdsrg_smart_s.cc

```diff
@@ -142,7 +142,7 @@ double MRDSRG::smart_s_davg_min_delta1() {
     }
     double davg_sum = std::accumulate(davg.begin(), davg.end(), 0.0);
     std::transform(davg.begin(), davg.end(), davg.begin(),
-                   std::bind1st(std::multiplies<double>(), 1.0 / davg_sum));
+                   [&](double x) { return x / davg_sum; });
```

Comment:
@lcyyork : can you check that this is the correct translation of the `bind1st` call here? `bind1st` is not supported in C++17 and a lambda should be the simplest solution.

Reply:
@fevangelista Thanks for the modification. This should be good.

Reply:
Great. If you have chance please review and comment/approve

Reply:
I do not have any problem so far.